sycl/include/CL/sycl/detail/pi.h

```diff
@@ -629,17 +631,12 @@ pi_result piextGetDeviceFunctionPointer(
 //
 // Context
 //
-pi_result piContextCreate(
-  const cl_context_properties * properties, // TODO: untie from OpenCL
-  pi_uint32         num_devices,
-  const pi_device * devices,
-  void (*           pfn_notify)(
-    const char * errinfo,
-    const void * private_info,
-    size_t       cb,
-    void *       user_data),
-  void *            user_data,
-  pi_context *      ret_context);
+pi_result piContextCreate(const pi_context_properties *properties,
+                          pi_uint32 num_devices, const pi_device *devices,
+                          void (*pfn_notify)(const char *errinfo,
+                                             const void *private_info,
+                                             size_t cb, void *user_data),
+                          void *user_data, pi_context *ret_context);
```

Comment:
I guess this is properly clang-formatted, but could you please preserve original code style? Just for unification purposes

Reply:
I am not sure how to pass the automatic clang-format checker though as it pointed this out as requiring clang-formatting?

Reply:
Oh, let's ask @bader about our policy on clang-format-check

Reply:
PS: @AlexeySachkov and you are completely right pointing this out - I just blindly applied the clang-format without taking the context into account.

Reply:
@AlexeySachkov, we have policy to follow LLVM coding style (including formatting style), so I think it's a reasonable change.
The question is "why pi.h doesn't follow LLVM formatting style"?
I would rather apply clang-format to the whole header file to unify the style.

Reply:
I will push two commits, one to reformat `pi.h` to LLVM coding style and one with the actual fixes of this PR on top. Currently running tests locally.

Reply:
Pushed the reformatting of `pi.h` and the fixes.